frontend/catalyst/pennylane_extensions.py

```diff
@@ -1057,7 +1065,8 @@ class QJITDevice(qml.QubitDevice):
         "Hamiltonian",
     ]
 
-    def __init__(self, shots=None, wires=None):
+    def __init__(self, shots=None, wires=None, backend=None):
+        self.backend = backend if backend else "default"
```

Comment:
It's more letting the runtime use the "default" (and available) device.

Reply:
How would this happen though, doesn't the user _need_ to specify a backend device when using PL?

Reply:
This is just for internal use cases (testing) with creating `QJITDevice` directly without specifying amy device. This can be removed if not necessary.